buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -195,8 +195,8 @@
     "BuildConfiguration": {
       "value": "Release"
     },
-    "PB_BuildArguments": {
-      "value": "/p:TargetArchitecture=$(PB_TargetArchitecture) /p:ConfigurationGroup=$(BuildConfiguration) $(PB_AdditionalBuildArguments)"
+    "PB_CommonMSBuildArgs": {
```

Comment:
Since everything is turning into portable, what if we flipped this?  The default should be portable=true.  And only in places where that isn't true, set it to false.

Maybe, let's merge as is to get the build unblocked.  Then come back through and turn all build defs to portable=true by default.

thoughts?
/cc @gkhanna79